configure.ac

```diff
@@ -81,6 +81,7 @@ MUTTER_PC_MODULES="
    xrender
    x11-xcb
    xcb-randr
+   xcb-res
```

Comment:
We are going to need a small PR for the debian branch as well, that adds a build dependency on `libxcb-res0`